src/invidious/comments.cr

```diff
@@ -60,8 +60,6 @@ def fetch_youtube_comments(id, cursor, format, locale, thin_mode, region, sort_b
   case cursor
   when nil, ""
     ctoken = produce_comment_continuation(id, cursor: "", sort_by: sort_by)
-    # when .starts_with? "Ug"
-    #   ctoken = produce_comment_reply_continuation(id, video.ucid, cursor)
```

Comment:
This has been commented out for two years https://github.com/iv-org/invidious/commit/7b53b6bfeff1670c1c5d3dd299b7ed6162a5e3c0